content/post/072-js-ipfs-0.39.md

```diff
@@ -0,0 +1,140 @@
+---
+date: 2019-10-24
+url: 071-js-ipfs-0-39
+title: js-ipfs 0.39.0 released
+author: Alex Potsides
+---
+
+# 🔦 Highlights
+
+> Configure your node with preset profiles! Limit outgoing browser requests! Start your node from fresh with one command!
```

Comment:
Alan tends to do a truly delightful tweet embed. Totally optional - but would expand on the highlights a bit. =]

Reply:
I've reformatted it to be more similar to previous js-IPFS posts